server/app/views/applicant/EditTiClientView.java

```diff
@@ -134,12 +138,24 @@ private DivTag renderMainContent(
     else {
       return div()
           .with(
-              ApplicationBaseView.requiredFieldsExplanationContent(messages),
+              requiredFieldsExplanationContent(messages),
               renderAddOrEditClientForm(
                   tiGroup, optionalAccountModel, request, tiClientInfoForm, messages));
     }
   }
 
+  /**
+   * Renders "Note: Fields marked with a * are required."
+   *
+   * @param messages the localized {@link Messages} for the current applicant
+   * @return PTag containing requiredness text.
+   */
+  private static PTag requiredFieldsExplanationContent(Messages messages) {
```

Comment:
(note) Only used once so moved it here in order to delete the other obsolete class.